packages/code-du-travail-nlp/api/__init__.py

```diff
@@ -1,31 +1,43 @@
 from flask import Flask
 from api.suggest import add_suggest
```

Comment:
Tu devrais importer les sous-modules de maniere pythonique
```suggestion
from .suggest import add_suggest
```